sktime/forecasting/ets.py

```diff
@@ -200,6 +223,11 @@ def __init__(
         additive_only=False,
         ignore_inf_ic=True,
         n_jobs=None,
+        method="lbfgs",
+        skip_hessian=True,
+        approx_centered=False,
+        approx_complex_step=True,
+        return_raw=False,
```

Comment:
I am wondering where this params are used? It looks you just added them but they are actually not used, are those params actually fit params from `statsmodels`? Then we need to give them to fit

Reply:
I was not sure about some of these as well. I'd like to discuss them.
1. `skip_hessian` is in the `fit` parameter of `super().fit()` in [ets.py](https://github.com/statsmodels/statsmodels/blob/main/statsmodels/tsa/exponential_smoothing/ets.py) of statsmodel. The `super` here is class [MLEModel](https://github.com/statsmodels/statsmodels/blob/main/statsmodels/tsa/statespace/mlemodel.py). 
2. `method` is in the `fit` parameter of `MLEModel` class as well, but somehow they are not carried over to [ETSModel](https://github.com/statsmodels/statsmodels/blob/main/statsmodels/tsa/exponential_smoothing/ets.py)
3. `approx_centered` and `approx_complex_step` are parameters of method `hessian` in `ETSModel`, they are not part of fit(). Since the issue wants parameters outside of the contructor and `fit` method, I added these. 
4. `return_raw` is a parameter of method `smooth`, also in `ETSModel`.

I realize these are not called in sktime, but how should we best incorporate them? Otherwise, they can be discarded.

Reply:
Please only add params that are exposed in `statsmodels` to the user either in constructor or in `fit`. Pls make sure that you have all params from `fit`, those are the ones that were actually missing at least in some cases

Reply:
In that case, there is no change needed. All parameters from `fit` and constructor were accounted for. I will undo the commit